R/generate_data.R

```diff
@@ -1035,27 +1035,38 @@ genSurv <- function(dtName, survDefs, digits = 3,
   
   for (i in (seq_along(events))) {
     
-    nlogu <--log(stats::runif(nrow(dtSurv), min = 0, max = 1))
+    nlogu <- -log(stats::runif(nrow(dtSurv), min = 0, max = 1))
```

Comment:
An important space :D

Reply:
@assignUser Very important. Interestingly, data.table has literally implemented its own double-dot notation that does virtually the same thing ours does. There's actually no need to parse it, data.table does it for you. But, I just used ours for consistency sake.

Reply:
Oh neat I did not know that, dt was the inspiration for the `..` prefix  :D